servicetalk-http-utils/src/test/java/io/servicetalk/http/utils/TimeoutHttpRequesterFilterTest.java

```diff
@@ -72,6 +74,10 @@ private static Single<StreamingHttpResponse> applyFilter(TimeoutHttpRequesterFil
         when(connection.request(any())).thenReturn(responseSingle);
 
         StreamingHttpRequester requester = filterFactory.create(connection);
-        return requester.request(mock(StreamingHttpRequest.class));
+        StreamingHttpRequest request = mock(StreamingHttpRequest.class);
+        ContextMap requestContext = mock(ContextMap.class);
+        when(request.context()).thenReturn(requestContext);
+        when(requestContext.getOrDefault(HTTP_EXECUTION_STRATEGY_KEY, strategy)).thenReturn(strategy);
```

Comment:
Would it make sense to add a test case which asserts the newly added functionality (so one where HTTP_EXECUTION_STRATEGY_KEY is present and applied correctly?)

Reply:
I made an attempt but then realized that with mocks it won't really make any difference because it's anyway have to go through `requestContext.getOrDefault(...)`. So, having the value for `HTTP_EXECUTION_STRATEGY_KEY` or fallback to default results in the same line for mock

Reply:
👍